src/ansys/fluent/core/services/datamodel_se.py

```diff
@@ -1253,23 +1285,32 @@ def create_command_arguments(self, command: str) -> str:
             self.rules, convert_path_to_se_path(self.path), command
         )
 
-    def add_on_child_created(self, child_type: str, cb: Callable) -> EventSubscription:
+    def add_on_child_created(
```

Comment:
This is the user-facing function and has identical implementation in PyFluent and PyConsole. In the user-facing function the callback takes an object of PyFluent's PyMenu class in both PyFluent and PyConsole. The user-facing workflow classes in PyFluent wraps further this function.